nntrainer/models/neuralnet.cpp

```diff
@@ -396,9 +396,9 @@ void NeuralNetwork::saveModel() {
     << "model file not opened, file path: " << save_path
     << " reason: " << strerror(errno);
 
-  for (auto iter = model_graph.cbegin(); iter != model_graph.cend(); iter++)
-    (*iter)->getObject()->save(model_file);
-
+  for (auto iter = model_graph.cbegin(); iter != model_graph.cend(); iter++) {
```

Comment:
not directly related to this PR, but now this can be simplified to 
```
for (auto const &lnode : std::as_const(model_graph)) {
```

Reply:
Thanks